web_source/_quarto.yml

```diff
@@ -18,44 +18,49 @@ book:
   repo-actions: [edit, issue]
   chapters:
     - index.qmd
-    - acknowledgements.qmd
-    - acronyms.qmd
-    - cite.qmd
-    - disclaimer.qmd
+    - pref-acknowledgements.qmd
```

Comment:
Filenames become part of the url so whilst I agree that having a prefix would be useful for organisational purpose related to the source of the page, it probably doesn't help with search engine optimisation / findability of pages to website users.